packages/manager/apps/dedicated/client/app/otrs/otrs.html

```diff
@@ -291,7 +291,7 @@ <h1 data-translate="otrs_title"></h1>
             </tbody>
             <tbody data-ng-show="!loaders.tickets && tickets.ids.length">
                 <tr
-                    data-ng-repeat="ticket in tickets.detail | orderBy:'creationDate':true track by ticket.ticketNumber"
+                    data-ng-repeat="ticket in tickets.detail | orderBy:'creationDate':true track by ticket.ticketNumber+$index"
```

Comment:
added a fix here because there is a silent error in production (duplicate index in repeater)